capabilities/documentation-quality.md

```diff
@@ -1,51 +1,54 @@
 # [Documentation Quality](https://dora.dev/devops-capabilities/process/documentation-quality/)
 
-Documentation quality refers to the accuracy, completeness, and accessibility of internal documentation, enabling teams to effectively collaborate, make informed decisions, and deliver high-quality software quickly and reliably. By investing in documentation quality, developers can reduce errors, improve productivity, and enhance overall performance.
+
+_Documentation_ refers to written descriptions of internal decisions, processes, or policies. A team of software developers regularly creates and maintains documentation on a range of things, including technical design details, simple how-to cheat sheets, product requirements, pitfalls to be aware of in the tech stack, and testing plans. And whenever changes are made in code, it must be reflected in documentation. 
+
+_Documentation quality_ refers to the accuracy, clarity, completeness, and accessibility of internal documentation. Excellent documentation enables teams to effectively collaborate, make informed decisions, and deliver high-quality software quickly and reliably.
 
 ## Nuance
 
 This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
 
 ### Importance of Clear and Findable Documentation
 
-Clear and findable documentation is essential for effective knowledge sharing and collaboration within teams. Without it, teams may struggle to understand existing systems, leading to a decrease in development speed and reliability.
+It's not enough for documentation to exist; it must also be clear and findable. Clear and findable documentation is essential for effective knowledge sharing and collaboration within teams. Without it, teams may struggle to understand existing systems, leading to a decrease in development speed and reliability. Misleading documentation can be worse than missing documentation.
 
 ### Reliability and Maintenance of Documentation
 
-Documentation must be actively maintained and updated to ensure it remains relevant and accurate. This requires a deliberate effort to review and refine documentation regularly. Sometimes misleading documentation can be worse than missing documentation.
+Once documentation is created, the work is not done. Documentation must be actively maintained and updated to ensure it remains relevant, reliable, and accurate. This requires a deliberate effort by developers to review and refine documentation regularly. 
```

Comment:
Suggest clarifying (even broadly) what we mean by "regularly." (e.g., "For some teams, this could mean a documentation audit every 4 weeks. For other teams, this could mean real-time updates to code changes.")

Reply:
Yeah that's a tough one because every situation is different. Ultimately the answer is probably gonna be different for every team. The important outcome is that documentation reflects reality. If reality becomes out of step with documentation every day, thin that team might need to do those audits more frequently than a team who manages the system where their documentation becomes out of step with reality once a month.

Reply:
There's probably a missing nuance here around finding the right balance between writing documentation that's low-level enough to point people in a specific direction and writing documentation that's a little more abstract but isn't likely to change over time. Can you take a stab at putting something like that together?

Reply:
See what you think of my edit in 11/14 commit.